src/frontends/paddle/src/op/where.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs where(const NodeContext& node) {
+    const auto condition_node = node.get_input("Condition");
+    const auto x_node = node.get_input("X");
+    const auto y_node = node.get_input("Y");
```

Comment:
as we synced, Paddle framework has bug in cases that input("X") and input("Y") have different shape, e.g.
it will throw error for:
_import paddle
x = paddle.to_tensor([0.9383, 0.1983, 3.2, 1.2])
y = paddle.to_tensor([1.0, 1.0, 1.0])
out = paddle.where(x>1, x, y)_
so, we should not worry about these cases' op conversion. but for double insurance,  still added an 'PADDLE_OP_CHECK' for these cases.

Reply:
>> PADDLE_OP_CHECK(node, x_shape.compatible(y_shape), "shape x should be equal to shape y!");
Better change the comment to "shape x should be compatible to shape y"
And also please add unit test for it (disable it by default) and use a ticket to track.